server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewPeriodServicesImpl.java

```diff
@@ -96,6 +127,37 @@ public ReviewPeriod update(@NotNull ReviewPeriod reviewPeriod) {
             throw new BadArgException(String.format("Invalid status transition from %s to %s", currentStatus, newStatus));
         }
 
+        if (currentStatus != ReviewStatus.AWAITING_APPROVAL &&
+                newStatus == ReviewStatus.AWAITING_APPROVAL) {
+            notifyRevieweeSupervisorsByReviewPeriod(reviewPeriod.getId(), reviewPeriod.getName());
+        }
+
+
         return reviewPeriodRepository.update(reviewPeriod);
     }
+
+    private void notifyRevieweeSupervisorsByReviewPeriod(UUID reviewPeriodId, String reviewPeriodName) {
+        if (environment.getActiveNames().contains(Environments.LOCAL)) return;
+        Set<ReviewAssignment> reviewAssignments = reviewAssignmentRepository.findByReviewPeriodId(reviewPeriodId);
+        Set<UUID> revieweeIds = reviewAssignments.stream().map(ReviewAssignment::getRevieweeId).collect(Collectors.toSet());
+        Set<UUID> supervisorIds = new HashSet<>(memberProfileRepository.findSupervisoridByIdIn(revieweeIds));
+        if (supervisorIds.isEmpty()) {
+            LOG.info(String.format("Supervisors not found for Reviewees %s", revieweeIds));
+            return;
+        }
+
+        Set<String> supervisorIdsToString = supervisorIds.stream().map(UUID::toString).collect(Collectors.toSet());
+        List<String> supervisorEmails = memberProfileRepository.findWorkEmailByIdIn(aesKey, supervisorIdsToString);
+
+        // send notification to supervisors
+        String emailContent = constructEmailContent(reviewPeriodId, reviewPeriodName);
+        emailSender.sendEmail(null, null, "Review Assignments Awaiting Approval", emailContent, supervisorEmails.toArray(new String[0]));
+    }
+
+    private String constructEmailContent (UUID reviewPeriodId, String reviewPeriodName){
+        String emailHtml = "<h3>Review Assignments for Review Period '" + reviewPeriodName + "' are ready for your approval.</h3>";
+        emailHtml += "<a href=\"" + webAddress + "/feedback/reviews?period=" + reviewPeriodId + "\">Click here</a> to review and approve reviewer assignments in the Check-Ins app.";
+        return emailHtml;
```

Comment:
Can use text blocks again to remove the requirement to escape double quotes

```suggestion
        return """
                <h3>Review Assignments for Review Period '%s' are ready for your approval.</h3>\
                <a href="%s/feedback/reviews?period=%s">Click here</a> to review and approve reviewer assignments in the Check-Ins app.""".formatted(reviewPeriodName, webAddress, reviewPeriodId);
```